lms/templates/learner_dashboard/program_card.underscore

```diff
@@ -1,30 +1,53 @@
 
+<div class="text-section">
+    <h3 id="program-<%- id %>" class="title"><%- gettext(name) %></h3>
+    <div class="meta-info">
+        <div class="organization"><%- orgList %></div>
+        <div class="category">
+            <span class="category-text"><%- gettext(type) %></span>
+            <span class="xseries-icon" aria-hidden="true"></span>
+        </div>
+    </div>
+  <% if (progress) { %>  
+    <p class="certificate-status"><%= interpolate(
+        gettext('You have earned Verified Certificates in %(completed_courses)s of the %(total_courses)s courses so far.'),
```

Comment:
@marcotuts this is a good point that @rlucioni made. Is it okay to drop the word 'Verified' from the messaging here?